README.md

```diff
@@ -0,0 +1,55 @@
+# JS Library
+
+![homepage](https://i.imgur.com/qy75hBv.png)
+
+---
+
+In this project we made a webapp where you can add a list of the books you've read
+
+---
+
+## Local installation
+
+1. Clone this repo
+```
+$ git clone git@github.com:YoseptF/library.git
+``` 
```

Comment:
isn't it `git clone https://github.com/YoseptF/library.git` ??

Reply:
They are both valid, the one we used in the README.md is this one
![image](https://user-images.githubusercontent.com/44252641/78941754-e0d86480-7a7d-11ea-8da0-73b05630cb2b.png)
and you are using the https version
![image](https://user-images.githubusercontent.com/44252641/78941773-eafa6300-7a7d-11ea-93ae-ee60d6242445.png)